tensorflow/python/keras/saving/save_test.py

```diff
@@ -71,6 +71,13 @@ def test_save_format_defaults(self):
     save.save_model(self.model, path)
     self.assert_saved_model(path)
 
+  @test_util.run_v2_only
+  def test_save_format_defaults_pathlib(self):
+    if sys.version_info >= (3, 6):
```

Comment:
Let's change this to:
```
if sys.version_info < (3, 6):
   self.skipTest('pathlib is only available for python version >= 3.6')
```

This will make sure the test result show as skipped, rather than pass (but didn't verify any under the hood).

Reply:
Thank you for the suggestion. Some other tests that was already in the file have similar situation. Should I also fix those?

Reply:
Yes. Please do that.